tutorial/const_keyword.md

```diff
@@ -0,0 +1,86 @@
+# Const keyword
+
+In case if you want to make sure that an argument will not change inside its function, Pyccel provides a `const` keyword which is a qualifier of the data type of the variable and which will be converted to its equivalents in C/Fortran, Here is a simple example of its usage:
```

Comment:
I suggest using `the target language` instead of `C/Fortran`. It means one less thing to change if/when we add an additional language